modules/proxy.js

```diff
@@ -1,21 +1,48 @@
-module.exports = async ({msg,bot,members,cfg}) => {
+const db = require("./db");
+
+module.exports = async ({msg,bot,members,cfg,automember}) => {
     if(msg.channel.guild && (!msg.channel.permissionsOf(bot.user.id).has("readMessages") || !msg.channel.permissionsOf(bot.user.id).has("sendMessages"))) return;
 	if(members[0] && !(msg.channel.type == 1)) {
+		let matchPhraseEndsWithAuto = new RegExp("^.*-"+ cfg.prefix + "auto$");
+		let matchLineEndsWithAuto = new RegExp("\\s*-"+ cfg.prefix + "auto$");
+		let matchAnyLineEndsWithAuto = new RegExp("\\s*-"+ cfg.prefix + "auto$", "m");
+
+
 		let clean = msg.cleanContent || msg.content;
 		clean = clean.replace(/(<a?:.+?:\d+?>)|(<@!?\d+?>)/,"cleaned");
+		//for the cleaned array, filter instances of -prefix!auto at the end of each line
+		clean = clean.replace(matchAnyLineEndsWithAuto, "");
 		let cleanarr = clean.split("\n");
 		let lines = msg.content.split("\n");
+
+		//these variables store the member which was used with tul-auto
+		//as well as if tul!auto was used (used to write to database when proxying message)
+		let setAutoProxy = false;
+		let setAutoProxyMember = null;
+
 		let replace = [];
 		let current = null;
+		//this flag identifies if the member that was picked was picked due to auto proxy instead of bracket matching
+		let usingautomember = false;
```

Comment:
This variable doesn't seem to be used anywhere?

Reply:
removed